### DISCUSSION TITLE
Interface Needs

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#390] [Ideas]
Interface Needs

Hello, I think there are some features that are needed during use.

1- When the moderator is connected from the mobile screen and the user joins the conversation, the users' screen appears in full screen to give approval. When the confirm button is pressed, the users screen should close. Or when the screen is turned on for Approval, the width of the screen should be around 80%, and users should be able to hide the screen when the background is touched. It is better to close it automatically once confirmed. Instead of opening the user screen for user confirmation, it is much better to open a modal or small notification screen.

2- If connection via microphone is selected when entering the room, the automatic camera selection screen should open. If the camera is shared, the microphone selection screen should open. There should be a cancel button next to share on both screens. Between just listen and the microphone option, or when that screen is closed, a direct connection to the room is established. I can't see any difference in the connection to the room between these 3 options. In all three, it is necessary to give camera or microphone permission.

3- The default language should be set when creating the room. Although the browser is Turkish, Plugnmeet can define it in English and open it in English.

4- When the Camera or Microphone selection screen opens, if the list is empty, a small warning may be given at the bottom. If the camera is open in a different application or tab, plugnmeet cannot access the camera list normally and it appears empty. Give permission to access the camera or another application is currently using the camera.

5- CODE information is displayed on the screen when the room is closed or other notifications. The information message is sufficient, I think it is not necessary for the end user to display the notification code.

6- There should be an exit button in the submenu to quickly exit the conversation.

7- It would be very good for simplification if all visible sections could be closed parametrically while creating the room. For example, the survey section can be closed, the raise hand button can be turned off, etc.

I may seem to be asking too much, but all of this is necessary for the improvement, simplification and ease of use of the system, generalization of use. I am writing this so that you can make a very useful system even more useful.

**Comment by jibon57** [comment]
Thanks for the suggestions. In the future we may look on those. But we want to keep the interface as simple as now. Like separate microphone & webcam selection. We don't want to make those into one.

**Comment by oksuzgil** [comment]
Thank you for your attention. Although it seems like an individual request, I am sharing what I encountered as a result of field use tests. Just as the user interface is important, the user experience is also very important, you know.

I want you to know that I am always ready to share my experiences by providing live and active meetings.
